File: testdata/python3/book/create10.py
#!/usr/bin/env python
# -*- coding:utf-8 -*-
"""
function to say hi in python
"""

import json

def handler(event, context):
    obj = {
        "name": "baetyl",
        "project": "github"
    }
    resp = {
        "isBase64Encoded": False,
        "statusCode": 200,
        "headers": {
            "X-Custom-Header": "headers"
        },
        "body": json.dumps(obj)
    }
    return resp

File: testdata/node10/book/create10.js
#!/usr/bin/env node

exports.handler = (event, context, callback) => {
    let obj = {
        "name": "baetyl",
        "project": "github"
    }
    callback(null, {
        "isBase64Encoded": false,
        "statusCode": 200,
        "headers": {
            "X-Custom-Header": "headers"
        },
        "body": JSON.stringify(obj)
    });
}